app/models/footer.rb

```diff
@@ -0,0 +1,54 @@
+require "yaml"
+
+# Model wrapping a blob of YAML data describing settings related to the global site footer.
+class Footer
+  class Link
+    attr_reader :url, :text
+    def initialize(url, text)
+      @url = url
+      @text = text
+    end
+  end
+
+  attr_reader :data
+
+  def self.load(file = Rails.root.join("_settings", "footer.yml"))
+    data = YAML.safe_load File.read(file), fallback: {}
+    new data
+  end
+
+  def initialize data
```

Comment:
[nitpick] as a matter of style, I prefer using parenthesis in method definitions. I'm actually a little surprised this passes the linter.

Reply:
Wow, I definitely didn't do that on purpose!